examples/specs/normalized/airport_connections_normalized.vl.json

```diff
@@ -90,5 +90,6 @@
   ],
   "width": 900,
   "height": 500,
-  "config": {"view": {"stroke": null}}
+  "config": {"view": {"stroke": null}},
+  "autosize": {"type": "pad"}
```

Comment:
Do we need to add this or is it already the default in Vega? It would be good to reduce churn.

Reply:
Done! Thank for the suggestion.